sdk/ai/ai-projects/samples-dev/agents/agentsAzureFunctions.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to use azure function agent operations from the Azure Agents service
+ *
+ * @summary demonstrates how to use azure function agent operations from the Azure Agents service
+ *
+ */
+
+import { AIProjectsClient, ToolUtility } from "@azure/ai-projects";
+import { DefaultAzureCredential } from "@azure/identity";
+
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
Not blocking but we are moving to the new recommended way:

```suggestion
import  "dotenv/config";
```

Reply:
@ZachhK , Please address them in separate  PR